api/src/glfs-fops.c

```diff
@@ -605,6 +605,17 @@ setup_entry_fopat_args(uuid_t gfid, dict_t **xattr_req, loc_t *loc)
     return ret;
 }
 
+/* This function is of use when filepath is not present to
+ * resolve and to fill loc.
+ */
+static void
+fd_to_loc(struct glfs_fd *pglfd, loc_t *loc)
```

Comment:
nit: Change variable name to `glfd`. Move this function to `glfs-resolve.c`?

Reply:
Ack will move it

Reply:
done!